### DISCUSSION TITLE
How to turn on spell check automatically in `NeogitCommitMessage`?

### DISCUSSION THREAD

**Discussion by Jaehaks** [NeogitOrg/neogit#1477] [Q&A] [answered]
Hi, 

When I open commit message buffer, `spell` option always turns off. 
autocmd for this filetype cannot work. how to change this?

**Comment by CKolkey** [comment, marked as answer]
Oh, I disable it in all buffers. It would make sense to enable spell checking in messages though. I'll change that.

**Reply by CKolkey** [reply]
https://github.com/NeogitOrg/neogit/pull/1478

**Reply by Jaehaks** [reply]
What a quick fix!
Thanks:)

**Reply by CKolkey** [reply]
Luckily it was very very simple haha. Glad you pointed it out
